packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -281,17 +281,31 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>> extend
           ev.preventDefault();
           ev.stopPropagation();
         }
+        this._updateActiveDescendant();
         break;
 
       case KeyCodes.down:
         if (this.suggestionsControl && this.suggestionsControl.handleKeyDown(keyCode)) {
           ev.preventDefault();
           ev.stopPropagation();
         }
+        this._updateActiveDescendant();
         break;
     }
   };
 
+  private _updateActiveDescendant(): void {
+    if (this.props.inputElement) {
+      let activeId: string;
+      if (this.currentSelectedSuggestionIndex >= 0) {
+        activeId = `sug-${this.currentSelectedSuggestionIndex}`;
```

Comment:
Is there a reason for using `currentSelectedSuggestionIndex` instead of `selectedHeaderIndex` or `selectedFooterIndex`?